simplestore/src/main/java/com/uber/simplestore/impl/SimpleStoreImpl.java

```diff
@@ -15,16 +15,12 @@
  */
 package com.uber.simplestore.impl;
 
-import android.content.Context;
 import android.util.Log;
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.util.concurrent.Futures;
 import com.google.common.util.concurrent.ListenableFuture;
 import com.google.common.util.concurrent.MoreExecutors;
-import com.uber.simplestore.NamespaceConfig;
-import com.uber.simplestore.SimpleStore;
-import com.uber.simplestore.SimpleStoreConfig;
-import com.uber.simplestore.StoreClosedException;
+import com.uber.simplestore.*;
```

Comment:
Switch back to explicit import

Reply:
Wasn't sure if that was done by IDE or spotless, so that's why I left it 🤷‍♂️ 
Switched back now